File: src/app/interfaces/receipt-history-dto.ts
export interface ReceiptHistoryDTO {
  receiptId: string;
  receiptNumber: string;
  receiptDate: Date;
  merchantName: string;
  merchantAddress: string;
  customerName: string;
  customerAddress: string;
  customerPhone: string;
  subTotalAmount?: number;
  totalAmount?: number;
  taxAmount?: number;
  otherCharge?: number;
  statusId: number;
  statusName: string;
  serviceChargeAmount?: number;
}

File: API/FunctionAppDoc2Data/Models/ReceiptHistoryDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FunctionAppDoc2Data.Models;
public class ReceiptHistoryDTO
{
    public Guid ReceiptId { get; set; }
    public string ReceiptNumber { get; set; }
    public DateTime ReceiptDate { get; set; }
    public string MerchantName { get; set; }
    public string MerchantAddress { get; set; }
    public string CustomerName { get; set; }
    public string CustomerAddress { get; set; }
    public string CustomerPhone { get; set; }
    public decimal? SubTotalAmount { get; set; }
    public decimal? TotalAmount { get; set; }
    public decimal? TaxAmount { get; set; }
    public decimal? OtherCharge { get; set; }
    public int StatusId { get; set; }
    public string StatusName { get; set; }
    public decimal? ServiceChargeAmount { get; set; }
}
